airflow/api_connexion/openapi/v1.yaml

```
@@ -3159,6 +3182,18 @@ components:
             *New in version 2.5.0*
           nullable: true
 
+    DagPriorityParsingRequest:
+      description: Dag parsing requests.
+      type: object
+      properties:
+        fileloc:
+          type: string
+        id:
+          type: string
+          readOnly: true
+          description: The request ID.
+          nullable: true
```

```suggestion
    DagPriorityParsingRequest:
      description: Request to reparse DAG(s) in a file.
      type: object
      properties:
        fileloc:
          type: string
        id:
          type: string
          readOnly: true
          description: ID of the created reparse request.
          nullable: true
```

But from what I can tell, the endpoint does not actually return this?

---

Also I think the `id` field should be an implementation detail and not expose to the user. Since the value is derived from `fileloc`, we should only expose `fileloc` instead.